torch/csrc/jit/python/init.cpp

```diff
@@ -1299,10 +1300,25 @@ void initJITBindings(PyObject* module) {
           [](const c10::SymNode& node) {
             return node->nested_int();
           })
+      .def(
+          "nested_int_vec",
+          [](const c10::SymNode& node) {
+            TORCH_CHECK(node->is_nested_int());
+            return c10::get_nested_int_vec(node.get());
+          })
       .def(
           "nested_int_coeff",
           [](const c10::SymNode& node) {
+            TORCH_CHECK(node->is_nested_int());
             return node->nested_int_coeff();
+          })
+      .def(
+          "clone_nested_int_with_new_vec",
+          [](const c10::SymNode& node, const at::Tensor& vec) {
+            TORCH_CHECK(
+                false,
+                "Expect not to call clone_nested_int_with_new_vec on a "
+                "non-symbolic nested int");
```

Comment:
This is just a stub to provide a nice error message if someone does nested_int.node.clone_nested_int_with_new_vec when the nested int is not symbolic. 

> and also is it overridden somehow for symbolic nested ints?

Yeah, the real logic here is actually in SymNode. 

The only reason this clone needs to exists is so that source is properly propagated and fx graph properly captures everything, so no one should be calling the clone here.